framework/build.gradle

```diff
@@ -179,22 +179,18 @@ checkCompilerMessages {
   ]
 }
 
-task loaderTests(dependsOn: 'shadowJar', group: 'Verification') {
+task loaderTests(type: Exec, dependsOn: 'shadowJar', group: 'Verification') {
   description 'Run tests for the annotation class loader'
   dependsOn(compileTestJava)
   // TODO: this dependency on checker is a bit ugly.
   dependsOn project(':checker-qual').tasks.jar
   dependsOn project(':checker').tasks.assemble
-  doLast {
-    exec {
-      executable 'make'
-      args = [
-        '-C',
-        'tests/annotationclassloader/',
-        'all'
-      ]
-    }
-  }
+  executable 'make'
```

Comment:
Is there a reason not to use `commandLine` here, as is used elsewhere?

Reply:
switch to `commandLine`.